enumerable_methods.rb

```diff
@@ -194,7 +191,7 @@ def multiply_els(arr)
 p([nil, 2.5, 'hi', -7, true].my_each { |el| el })
 puts
 p 'My_each_with_index:'
-[nil, 2.5, 'hi', -7, true].my_each_with_index { |el, i| p("Element: #{el}, Index: #{i}") }
+p [nil, 2.5, 'hi', -7, true].my_each_with_index.to_a
 puts
 p 'My_select method:'
```

Comment:
## No debugging code
- No debugging instructions in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)
  - [ ] Remove all `p` statement in the entire file